code/__HELPERS/spatial_info.dm

```diff
@@ -1,8 +1,3 @@
-/turf
-	///what /mob/oranges_ear instance is already assigned to us as there should only ever be one.
-	///used for guaranteeing there is only one oranges_ear per turf when assigned, speeds up view() iteration
-	var/mob/oranges_ear/assigned_oranges_ear
```

Comment:
For reference, if you are trying to solve the ctrl-click problem then the following code won't trigger the navigate to definition behaviour:
```dm
///what /mob/oranges_ear instance is already assigned to us as there should only ever be one.
///used for guaranteeing there is only one oranges_ear per turf when assigned, speeds up view() iteration
/turf/var/mob/oranges_ear/assigned_oranges_ear
```

Reply:
Ok